praw/__init__.py

```diff
@@ -238,6 +238,13 @@ def config_boolean(item):
         self.https_proxy = (obj.get('https_proxy') or
                             os.getenv('https_proxy') or None)
         # We use `get(...) or None` because `get` may return an empty string
+
```

Comment:
I think we're using 2 different "defaults"  I meant I was making it the default in that if that line was removed from the ini for some reason, it will end up as True.  

I thought that you could leave other configuration options out of the ini completely and for most of them things would still work.  I did it this way to ensure if the option was removed from praw.ini it would default to the more secure option.